packages/headless/src/controllers/quickview/case-assist-headless-quickview.ts

```diff
@@ -0,0 +1,38 @@
+import {CaseAssistEngine} from '../../app//case-assist-engine/case-assist-engine';
+import {logQuickviewDocumentSuggestionClick} from '../../features/case-assist/case-assist-analytics-actions';
+
+import {
+  buildQuickviewCore,
+  QuickviewCoreOptions,
+  QuickviewCoreState,
+  QuickviewCoreProps,
+  QuickviewCore,
+} from '../core/quickview/headless-core-quickview';
+
+export interface CaseAssistQuickviewProps extends QuickviewCoreProps {}
```

Comment:
@samisayegh are those going to be documented properly by the tool ? I think so, but just making sure.

Reply:
I think the tooling will give an error, saying these interfaces need to be annotated with jsdoc.

I don't see a `case-assist` use-case defined in `doc-parser/use-cases` though. I'd suggest tackling the doc task for all case-assist controllers in a separate PR.